package.json

```diff
@@ -29,7 +29,7 @@
     "lint:watch": "nodemon --exec npm run lint",
     "build": "tsdown",
     "build:watch": "tsdown --watch",
-    "run-cli": "npm run build && node bin/typesync",
+    "run-cli": "npm run build -- --silent && node bin/typesync",
```

Comment:
This made it easier to see typesync's output, you don't care about the build if it passes.